adapters/Vungle/VungleAdapter/GADMAdapterVungleConstants.h

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-static NSString *const _Nonnull kGADMAdapterVungleVersion = @"6.4.6.0";
+static NSString *const _Nonnull kGADMAdapterVungleVersion = @"6.5.2.0";
```

Comment:
This PR is for `6.5.1.1` release, as per the CHANGELOG and title. Is this correct?

Reply:
@joshuajplagonera Hi joshuajplagonera, I'm really sorry for the confusing. Because I submitted a modification to support Adaptive and Smart Banners, which updated the kGADMAdapterVungleVersion from 6.5.1.1 to 6.5.2.0. I will update the CHANGELOG right now.

Reply:
@joshuajplagonera Hi joshuajplagonera, I have updated the CHANGELOG for 6.5.2.0 version. I'm really sorry for the confusing and thanks a lot for the reminder. Have a nice day. 😊

Reply:
@joshuajplagonera Hi joshuajplagonera, after we discussed, since we will release 6.5.2 SDK soon, we will hold this PR for Vungle AdMob Adapter 6.5.1.1. And we will create a new PR for Vungle AdMob Adapter 6.5.2 to support "Smart and Adaptive Banners". So I reset this PR to the previous 6.5.1.1 state. I'm really sorry for the confusing and have a nice day. 😊